helix-view/src/editor.rs

```diff
@@ -71,6 +81,11 @@ impl Default for FilePickerConfig {
             git_global: true,
             git_exclude: true,
             max_depth: None,
+            width_factor: 0.9,
+            height_factor: 0.9,
+            preview_width_factor: 0.5,
+            preview_min_width: 60,
+            file_min_width: 20,
```

Comment:
We should add a note that both of this should total up to 80.

Reply:
Ok. Out of curiosity, what is the reason? `preview-min-width = 25`, `file-min-width = 20` works, and `preview-min-width = 80`, `file-min-width = 30` works too.

Reply:
I thought that's why you put 80? Like previously I saw a number 80 being removed, and you replaced it with the sum of these two.

At the very least I think we should still make it look nice for 80x24 terminals.

Reply:
Done :-)